api/src/main/resources/marquez/db/migration/V66.2__alter_lineage_events_add_event_type.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TYPE EVENT_TYPE AS ENUM ('RUN_EVENT', 'DATASET_EVENT', 'JOB_EVENT');
```

Comment:
_Minor_: We standardized on not defining enums types in the DB layer but rather enforce them in the application layer. This way, it avoids a DB migration every time we (might) add a new event type.

Reply:
added extra commit for that: https://github.com/MarquezProject/marquez/pull/2661/commits/0a3f98aa19e5ea0d5aaeeba85ae6ae813f829fe1